### PR TITLE
[bitnami/matomo] add service account value

### LINKED ISSUE #11978: Matomo with Cloud SQL Proxy and Service Account
### Name and Version

bitnami/matomo 0.1.13

### What is the problem this feature will solve?

In order to connect to MySQL on GCP best practice is use of Cloud SQL Proxy sidecar, however this will require a service account attached to sidecar. For Matomo to connect to Cloud SQL we would need to have a sidecar running with an specific service account, having the sidecar was not a problem but I couldn't find a way to associate a service account to it.

### What is the feature you are proposing to solve the problem?

Adding a field for serviceAccountName that attaches to sidecars running

### What alternatives have you considered?

Deploying Cloud SQL Proxy as a deployment with a service and connect to that instead of sidecars

### PR DESCRIPTION
**Pull request by amirsaber**
### Description of the change
Additional value to attach service account to containers and sidecars

### Benefits
Connecting to services such as CloudSQL requires sidecars with specific service account. This allows such functionality.

### Applicable issues

  - fixes #11978

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/matomo/values.yaml [hunk 1]
```diff
@@ -182,6 +182,9 @@ initContainers: []
 ## @param sidecars Attach additional containers to the pod (evaluated as a template)
 ##
 sidecars: []
+## @param serviceAccount Attach serviceAccount to the pod and sidecars
```
Comment (fmulero) [inline bitnami/matomo/values.yaml:185]:
```suggestion
## @param serviceAccount Attach serviceAccountName to the pod and sidecars
```

**Review by fmulero — CHANGES_REQUESTED** [review]
Great! Thanks a lot @amirsaber, Could you have a look to my comments, there is a little typo

File: bitnami/matomo/values.yaml (hunk 1 above)
Comment (amirsaber) [reply bitnami/matomo/values.yaml]: Thanks @fmulero I figured both mention of `serviceAccount` should get updated

**Comment by fmulero** [comment]
Sorry @amirsaber I missed it from my radar, my fault.

Could you resolve the conflicts? I tried but it seems you have blocked it

Sorry for the inconvenience

**Review by fmulero — APPROVED** [review]
LGTM

**Pull request merged**